facts/anteater.md

```diff
@@ -0,0 +1,7 @@
+# Anteater
+
+- The anteater's stomach, similar to a bird's gizzard, has hardened folds and
+  uses strong contractions to grind the insects, a digestive process assisted
+    by small amounts of ingested sand and dirt.
+- The anteaters are more closely related to the sloths than they are to any
+    other group of mammals.
```

Comment:
I feel like this reads a little better, but I don't have a particularly strong opinion here.

```suggestion
- Anteaters are more closely related to sloths than they are to any
    other group of mammals.
```